<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1156</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRender_Flow_Field"/>
    <addaction name="actionRender_Boundary_Conditions"/>
    <addaction name="actionApply_Transform"/>
    <addaction name="action_Rasterize_Flow"/>
    <addaction name="separator"/>
    <addaction name="actionRender_Vn"/>
    <addaction name="actionRender_Thickness"/>
    <addaction name="actionRender_Vx"/>
    <addaction name="actionRender_Vy"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionApply_Transform"/>
  </widget>
  <action name="actionRender_Flow_Field">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Render Flow Field</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRender_Boundary_Conditions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Render Boundary Conditions</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionApply_Transform">
   <property name="text">
    <string>Apply Transform</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_Rasterize_Flow">
   <property name="text">
    <string>&amp;Rasterize Flow</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionRender_Vn">
   <property name="text">
    <string>Render Vn</string>
   </property>
  </action>
  <action name="actionRender_Thickness">
   <property name="text">
    <string>Render Thickness</string>
   </property>
  </action>
  <action name="actionRender_Vx">
   <property name="text">
    <string>Render Vx</string>
   </property>
  </action>
  <action name="actionRender_Vy">
   <property name="text">
    <string>Render Vy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
